truckersmp_cli/steamruntime_helper.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+"""
+Helper script for Steam Runtime.
+
+This script starts truckersmp-cli inject program
+and optionally wine-discord-ipc-bridge.
+"""
+
+import argparse
+import logging
+import os
+import subprocess as subproc
+
+from truckersmp_cli.main import setup_logging
+from truckersmp_cli.utils import start_wine_discord_ipc_bridge
+from truckersmp_cli.variables import Args
+
+
+def main():
+    """Start truckersmp-cli.exe and optionally winediscordipcbridge.exe."""
+    # options and arguments
+    desc = "Helper script for Steam Runtime."
+    arg_parser = argparse.ArgumentParser(description=desc)
+    arg_parser.add_argument(
+        "-v", "--verbose",
+        help="verbose output (none:error, once:info, twice or more:debug)",
+        action="count")
+    arg_parser.add_argument(
+        "--use-wine-discord-ipc-bridge",
+        help="use wine-discord-ipc-bridge",
+        action="store_true")
```

Comment:
What do you think of making this more modular to give an arbitrary amount of executables to this helper script which will then also started inside the container prefix.
I'm mainly looking at this trucksbook which seems somewhat popular. Obviously the executable has to be in a subfolder of `Dir.truckersmp_cli_data` so they are accessible from within the container or we also have to include all folders containing these executables.

To be more specific I'm thinking instead of `script --use-wine-discord-ipc-bridge` of something like this:
~~~ sh
script --exe="/path/to/wine-discord-bridge.exe" --exe="/path/to/asd.exe"
or
script "/path/to/wine-discord-bridge.exe" "/path/to/asd.exe"
~~~

Reply:
> script --exe="/path/to/wine-discord-bridge.exe" --exe="/path/to/asd.exe"

Seems good to me. I was considering passing `/path/to/winediscordipcbridge.exe` instead of a boolean (from `--use-wine-discord-ipc-bridge`) but it would be better to be able to handle multiple paths:
```py
arg_parser = argparse.ArgumentParser()
arg_parser.add_argument(
    "--executable", action="append", metavar="FILE",
    help="3rd party executable to start in Steam Runtime container (can be specified multiple times for multiple files)")
result = arg_parser.parse_args(["--executable", "/path/to/file1.exe", "--executable", "/path/to/file2.exe"])
print(result.executable)  # ["/path/to/file1.exe", "/path/to/file2.exe"]
result = arg_parser.parse_args(["--executable", "/path/to/file1.exe"])
print(result.executable)  # ["/path/to/file1.exe"]
```

BTW, note that this PR, 167, and 174 need to rebase against master *after merging #175*. Even if I rebase now, R1732 would be reported.